native_client/client.cc

```diff
@@ -135,16 +135,20 @@ main(int argc, char **argv)
   // Pass audio to DeepSpeech
   struct ds_result* result = LocalDsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
   if (result) {
-    printf("%s\n", result->string);
+    if (result->string) {
+      printf("%s\n", result->string);
+      free(result->string);
+    }
+
     if ((argc == 4) && (strncmp(argv[3], "-t", 3) == 0)) {
```

Comment:
Heh, just also noticed that this ought to be >=4 (or the check at the top of the file needs to be changed) :)